modules/shared/components/organisms/MiniSurveyView/MiniSurveyView.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import type { FC, ReactElement } from 'react';
+import Image from 'next/image';
+import MiniSurveyViewOptions from '../../molecules/MiniSurveyViewOptions/MiniSurveyViewOptions';
+import PostViewHeader from '../../molecules/PostViewHeader/PostViewHeader';
+import PostViewFooter from '../../molecules/postFooter/PostFooter';
+import type { IMiniSurveyView } from './IMiniSurveyView';
+
+const MiniSurveyView: FC<IMiniSurveyView.IProps> = ({ post }): ReactElement => {
+  const indexOfImage = 0;
+  return (
+    <div
+      className="bg-white p-m shadow-soft rounded-md grid-rows-1 gap-4 grid"
+      id={post.id}
+    >
+      <PostViewHeader
+        id={post.id}
+        date={new Date(post.created_at)}
+        name={post.user.name}
+        profilePic={post.user.profile_pic}
+        isHidden={post.is_hidden}
+        handlePostOptionsIconClick={(): boolean => {
+          return true;
+        }}
+      />
+      <div>
+        <h3 className="font-normal text-md">{post.caption}</h3>
+      </div>
+      {post.media[indexOfImage].url && (
+        <Image
```

Comment:
It seems that the Image component from Next does not inherit (as a child )the margin (space between) you set on the parent. 16px space should be set between the caption and the image.

Reply:
please add screen shot because for me it take the space as in the design